src/main/kotlin/app/revanced/patches/youtube/utils/sponsorblock/SponsorBlockPatch.kt

```diff
@@ -62,6 +62,9 @@ object SponsorBlockPatch : ResourcePatch() {
                 "ic_sb_edit.xml",
                 "ic_sb_logo.xml",
                 "ic_sb_publish.xml",
+ "ns_bg.xml",
```

Comment:
You should make it align like this
```suggestion
                 "ns_bg.xml",
                  "ic_sb_forward.xml",
                  "ic_sb_backward.xml",
```

Reply:
Sure, I just automated it, that's why it got that way. Fixed it.